functionaltests/src/test/java/specs/BaseFixture.java

```diff
@@ -221,15 +269,20 @@ protected Order createAndGetOrder(Payment payment, String currencyCode, String b
      *
      * @see #getVerifiedVisaPseudoCardPan()
      */
-    synchronized static protected String getUnconfirmedVisaPseudoCardPan() {
-      if (PSEUDO_CARD_PAN == null) {
-          String fetchedPseudoCardPan = fetchPseudoCardPan(getTestDataVisaCreditCardNo3Ds());
-          // TODO: remove the assert and logging if the future if everything goes fine
-          assert PSEUDO_CARD_PAN == null : "PSEUDO_CARD_PAN multiple initialization";
-          PSEUDO_CARD_PAN = fetchedPseudoCardPan;
-          LOG.info("Unconfirmed pseudocardpan fetched successfully");
-      }
-
+    protected String getUnconfirmedVisaPseudoCardPan() {
+        synchronized (BaseFixture.class) {
+            if (PSEUDO_CARD_PAN == null) {
+                String fetchedPseudoCardPan = fetchPseudoCardPan(getTestDataVisaCreditCardNo3Ds(),
```

Comment:
`final` for what? for the method? But is is overridden in second tenant tests fixture [`BaseTenant2Fixture#getUnconfirmedVisaPseudoCardPan()`](https://github.com/commercetools/commercetools-payone-integration/blob/954f1460d4bd2fa027f3e342913cd97eb081d452/functionaltests/src/test/java/specs/multitenancy/BaseTenant2Fixture.java#L83-L83)